package.json

```
@@ -34,6 +34,6 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "npm": "*"
+    "npm": "^7.0.0-beta.4"
```

This forces a beta installation upon all users. Could we do `< 7.0` instead? 
```suggestion
    "npm": "< 7.0"
```